tutorials/camkes-vm-crossvm/camkes-vm-crossvm.md

```diff
@@ -1,7 +1,7 @@
 <!--
-  Copyright 2017, Data61, CSIRO (ABN 41 687 119 230)
+  Copyright 2024, seL4 Project a Series of LF Projects, LLC.
 
-  SPDX-License-Identifier: BSD-2-Clause
+  SPDX-License-Identifier: CC-BY-SA-4.0
```

Comment:
Strictly speaking we can't change the license either, it's been released under BSD-2-Clause by CSIRO, and that's what it is. We should probably leave it as is.